### DISCUSSION TITLE
Usage for `cargo loco db create`

### DISCUSSION THREAD

**Discussion by mateusdeap** [loco-rs/loco#302] [Q&A] [answered]
First off, big thanks to @kaplanelad for adding this.

I do have some questions, however. I was indeed able to get a db by saying:

```
cargo loco db create
```

I noticed it only created the dev environment, but I'm wondering: how would this be done for all environments? What's exactly the usage for this command?

Finally, if I write up a bit of documentation on this, can I submit a PR to the website? (I promise this time I won't let this grow stale)

**Comment by kaplanelad** [comment, marked as answer]
@mateusdeap, you can run with the --environment flag. The create command loads first the configuration from the environment. 
When doing `cargo loco db create` -> the default is development. just pass --environment XXX to the command

```sh
cargo loco db create --environment test
```
